packages/fast-components-react-base/src/dialog/README.md

```diff
@@ -1,5 +1,5 @@
 # Dialog
-The *Dialog* component accepts children and can either be a modal, or a non-modal dialog. Modal dialogs prevent users from interacting with the rest of the page content, while non-modal dialogs allow users the ability to continue interacting with the page. The width and height of the dialog content area can be set by props. If no props are passed to set width or height, the dialog width will default to 640px and the height will default to 480px.
+The *dialog* component accepts children and can either be a modal, or a non-modal *dialog*. Modal *dialogs* prevent users from interacting with the rest of the page content, while non-modal *dialogs* allow users the ability to continue interacting with the page. The width and height of the *dialog* content area can be set by props. If no props are passed to set width or height, the *dialog* width will default to 640px and the height will default to 480px.
 
 ## Accessibility
-A *Dialog* must include *at least* one focusable control. When the dialog appears on the screen, keyboard focus should move to the first focusable element. While not required to be focused, a dialog should include a label. This can be applied as an `aria-label` using the `label` prop, or by referencing a child within the dialog with the `aria-labelledby` attribute using the `labelledBy` prop. Optionally, additional context can be provided to assistive technologies by including an `aria-describedby` attribute which can be applied using the `describedBy` prop. Learn more about dialog accessibility at [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role).
\ No newline at end of file
+A *dialog* must include *at least* one focusable control. When the *dialog* appears on the screen, keyboard focus should move to the first focusable element. While not required to be focused, a *dialog* should include a label. This can be applied as an `aria-label` using the `label` prop, or by referencing a child within the *dialog* with the `aria-labelledby` attribute using the `labelledBy` prop. Optionally, additional context can be provided to assistive technologies by including an `aria-describedby` attribute which can be applied using the `describedBy` prop. Learn more about *dialog* accessibility at [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role) and [W3C WAI-ARIA documentation](https://www.w3.org/TR/wai-aria-practices/#dialog_modal).
```

Comment:
I think this paragraph should only speak to the React props and not the underlying `aria` attributes. My feeling is that it should be sufficient accessibility guidance to document the abstraction and not the implementation details when we're handling the implementation details directly. eg.
"While not required to be focused, a *dialog* should include a label. The ID of that label should be provided to the *dialog's* labelledBy prop...."

Reply:
Ok, I basically removed anything that we handle anyway, and just left the links to a11y info... is this sufficient?